package.json

```diff
@@ -8,9 +8,9 @@
     "_test": "mocha -r ts-node/register tests/**/*.test.ts",
     "_build": "webpack --config webpack.config.js",
     "_start": "nodemon --watch src --delay 1 --exec 'ts-node' src/main.ts",
-    "test": "npm-run-all --serial json2ts _test",
+    "test": "NODE_ENV=test npm-run-all --serial json2ts _test",
```

Comment:
"debug": "NODE_ENV=development npm-run-all --serial json2ts _test",
How about adding one more?

Reply:
That's a good idea.